Add maps functions to WASM (#3596)
inspektor-gadget/inspektor-gadget: docs/gadget-devel/gadget-wasm-api-raw.md

Return value:
- 0 in case of success, 1 otherwise.

### eBPF Maps

#### `getMap(name string) uint32`

Get a handle to an existing eBPF map.

Parameters:
- `name` (string): Map's name

Return value:

> - (u32) Handle to the data source on success, 0 on error.

Reviewer: ```suggestion
- (u32) Handle to map on success, 0 on error.
```

#### `mapLookup(m uint32, keyptr uint64, valueptr uint64) uint32`

Lookup the map for a value corresponding to given key.

Parameters:
- `m` (u32): Map handle (as returned by `getMap`)

> - `keyptr` (u64): A `bufPtr` to data corresponding to key.

Reviewer: Should we describe what a bufPtr is on "Data types"?

Reviewer: The bufPtr concept is no described on this file before. I think we need to describe a bufPtr is a uint64 encoding length and size. It's somehow explained on the "Data types" section of this document, but btfPtr is not explicitly mentioned.

> - `keyptr` (u64): A `bufPtr` to data corresponding to key.

Reviewer: ```suggestion
- `keyptr` (u64): A `bufPtr` to key.
```

?

> - `valueptr` (u64): A `bufPtr` to a pointer of data corresponding to value. The pointed data will be updated to reflect the content of the map.

Reviewer: ```suggestion
- `valueptr` (u64): A `bufPtr` pointer to store the value of the map.
```

?
